src/svg.resize.js

```diff
@@ -113,11 +126,34 @@ class ResizeHandler {
 
   endResize (ev) {
     this.resize(ev)
-    this.events = []
+    this.eventType = ''
     off(window, 'mousemove.resize touchmove.resize', this.resize)
     off(window, 'mouseup.resize touchend.resize', this.endResize)
   }
-
+  rotate (e) {
+    const endPoint = this.el.point(getCoordsFromEvent(e))
+    const cx = this.el.cx()
+    const cy = this.el.cy()
+    const dx1 = this.startPoint.x - cx
+    const dy1 = this.startPoint.y - cy
+    const dx2 = endPoint.x - cx
+    const dy2 = endPoint.y - cy
+    const c = Math.sqrt(dx1 * dx1 + dy1 * dy1) * Math.sqrt(dx2 * dx2 + dy2 * dy2)
+    if (c === 0) return 0
+    let angle = Math.acos((dx1 * dx2 + dy1 * dy2) / c) / Math.PI * 180
+    if (endPoint.x < this.startPoint.x) {
+      angle = -angle
+    }
+    if (this.el.dispatch('rotate', { angle: angle, event: e, handler: this }).defaultPrevented) {
+      return
+    }
+    this.el.rotate(angle)
+  }
+  endRotate (ev) {
```

Comment:
endRotate and endResize is teh same when you made the changes from the other comment. So you dont need an extra method here. Simplification is key! :)